.github/workflows/code-quality.yml

```diff
@@ -18,22 +18,18 @@ jobs:
       # For repo checkout
       contents: read
     steps:
-      - uses: actions/checkout@v4
+      - name: Check out repository code
+        uses: actions/checkout@v4
 
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: "3.9"
+          python-version: "3.10"
 
       - name: Install dependencies
         run: |
-          python -m pip install "poetry==1.7.0"
-          poetry config virtualenvs.create false
-          poetry install --no-cache
-          poetry install --only-root
-
-      - name: Export PYTHONPATH
-        run: echo "PYTHONPATH=$(pwd)" >> $GITHUB_ENV
+          python -m pip install poetry
```

Comment:
**explain:** I don't think all of that was needed/was correct, so I've simplified it. Happy to hear if there's some historical/functional reason why it should be that way!

Reply:
This looks good, it's been a pain over time coming and finding a very out of date poetry version in ci so happy to trust latest!

Reply:
`poetry` not being managed .. in `poetry` is a fun problem.